Update some docs to Windows 11 (#3674)
MicrosoftDocs/windows-uwp: hub/android/xamarin-forms.md

To use this tutorial, you'll need the following:

> - Windows 10  or newer

Reviewer: ```suggestion
- Windows 11 or Windows 10
```

- [Visual Studio 2019: Community, Professional, or Enterprise](https://visualstudio.microsoft.com/downloads/) (see note)
- The "Mobile development with .NET" workload for Visual Studio 2019

> [!NOTE]

> > This guide will work with Visual Studio 2017, 2019 or 2022. If you are using Visual Studio 2017, some instructions may be incorrect due to UI differences between the two versions of Visual Studio.

Reviewer: ```suggestion
> This guide works with Visual Studio 2022, Visual Studio 2019, and Visual Studio 2017. If you're using Visual Studio 2017, some instructions might be incorrect due to UI differences between versions of Visual Studio.
```
